src/router.ts

```diff
@@ -51,8 +50,8 @@ authRouter.post("/login", adminLoginLimiter, AuthController.login);
 const apiKeyRouter = Router();
 baseRouter.use("/api-key", apiKeyRouter);
 apiKeyRouter.get("/", isAdmin, APIKeyController.getApiKeys);
-apiKeyRouter.post("/", isAdmin, APIKeyController.add);
-apiKeyRouter.delete("/", isAdmin, APIKeyController.delete);
+apiKeyRouter.post("/", isAdmin, APIKeyController.addApiKey);
+apiKeyRouter.delete("/", isAdmin, APIKeyController.deleteApiKey);
```

Comment:
I have to say that I don't like the addition of the class name in the method names. Because you already have the name in the class therefore it isn't needed in the method.
@94726 I don't know if you see it the same way.

Reply:
To be completely honest, I am somewhat pissed that half of the changes could be deemed as useless and time wasting.
So yes, I see it the same way, but it also doesn't matter, just a bit unnecessary.

Half of the codebase got reworked twice, because of a whim. Not to say that all of it was, but I think this whole "code cleaning" thing is a bit overdone, especially the opinionated changes...